tests/integration/datadict/submission/test_endpoints.py

```diff
@@ -909,6 +909,22 @@ def test_export_all_node_types_and_resubmit_tsv_with_empty_field(
     print(json.dumps(json.loads(resp.data), indent=4, sort_keys=True))
     assert resp.status_code == 200, resp.data
 
+def test_export_node_with_array_json(
+    client, pg_driver, cgci_blgsp, require_index_exists_off, submitter
+):
+    post_example_entities_together(client, submitter, extended_data_fnames)
+    with pg_driver.session_scope() as s:
+        case = pg_driver.nodes(md.Case).first()
+        print("LUCAAAAAA datadict")
```

Comment:
let's remove this debug log in `tests/integration/datadict/submission/test_endpoints.py` and `tests/integration/datadictwithobjid/submission/test_endpoints.py` 🙂

Reply:
Yes, sorry, totally forgot that.